src/Servers/HttpSys/src/AsyncAcceptContext.cs

```diff
@@ -117,14 +116,14 @@ private uint QueueBeginGetContext()
                 _requestContext.RequestId,
                 // Small perf impact by not using HTTP_RECEIVE_REQUEST_FLAG_COPY_BODY
                 // if the request sends header+body in a single TCP packet
-                (uint)HttpApiTypes.HTTP_FLAGS.NONE,
+                0u,
```

Comment:
Feels like a small loss in code quality. Is there not a "none" flag in the httpsys headers?

Reply:
No, there isn't. There isn't one defined natively either.
https://learn.microsoft.com/en-us/windows/win32/api/http/nf-http-httpreceivehttprequest
<html>
<body>
<!--StartFragment-->

Value | Meaning
-- | --
0 (zero) | Only the request headers are retrieved; the entity body is not copied.
HTTP_RECEIVE_REQUEST_FLAG_COPY_BODY | The available entity body is copied along with the request headers. The pEntityChunks member of the HTTP_REQUEST structure points to the entity body.
HTTP_RECEIVE_REQUEST_FLAG_FLUSH_BODY | All of the entity bodies are copied along with the request headers. The pEntityChunks member of the HTTP_REQUEST structure points to the entity body.

<!--EndFragment-->
</body>
</html>